styles.css

```diff
@@ -30,5 +30,5 @@ h2{
   color: aqua;
 }
 .social-media-links{
-  color: #ffd540;
+  color: #fac816;

```

Comment:
Kindly change the text color #fac816 to something more visible that will enable the user to read well. The best practice to outline text in HTML mockups is to make the text color white or any color that will come out well with the background. Doing this makes your website more fun to read on any info displaced. 

Many thanks